pom.xml

```diff
@@ -33,13 +33,13 @@ THE SOFTWARE.
         <relativePath />
     </parent>
 
-    <groupId>org.jenkins-ci.main</groupId> <!-- historical -->
-    <artifactId>ui-samples-plugin</artifactId> <!-- historical -->
+    <groupId>io.jenkins.plugins</groupId>
+    <artifactId>design-library</artifactId>
```

Comment:
I'd leave the GAV as-is, because changing it basically creates a new plugin from plugin manager's perspective which would result in `ui-samples-plugin` users not being able to update anymore and `design-library` installations could install the ui-samples-plugin.

Reply:
Hmm, given it hasn't been released in 8 years it's probably okay to change it, although likely we will want to change the package name as well.

Reply:
Would it work out to suspend ui-samples from the update center noting it's superseded by design-library?

Reply:
possibly